experimental/generation/generator/packages/library/src/dialogGenerator.ts

```diff
@@ -475,6 +475,45 @@ async function globalExamples(outDir: string, scope: any): Promise<object> {
     return examples
 }
 
+// Verify all $examples entries are entities or phrases
+async function verifyExamples(schema: any, feedback: Feedback): Promise<void> {
+    const allEntities = new Set<string>()
+    await walkJSON(schema, async (elt, _obj, path) => {
+        if (elt.$entities) {
+            for (const entity of elt.$entities) {
+                allEntities.add(entity)
+            }
+        }
+        return false
+    })
+    await walkJSON(schema, async (elt, _obj, path) => {
+        if (elt.$examples && !path?.endsWith('/items')) {
+            if (path) {
+                if (!elt.$entities) {
+                    feedback(FeedbackType.error, `${path} is missing $entities`)
+                } else {
+                    for (const [locale, examples] of Object.entries(elt.$examples)) {
+                        for (const entityOrPhrases of Object.keys(examples as object)) {
```

Comment:
What's the type of `elt.$examples`? Typescript should infer types correctly, so I'm curious why `as object` is necessary.

Reply:
We have not typed all of JSON schema so there is no strong type.  It is an object with keys that are locales that has an object value where the keys are entities and the values are either arrays of strings or an object with enum values each of which has an array of strings.